plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -2841,7 +2841,13 @@ function woocommerce_subcategory_thumbnail( $category ) {
 
 		if ( $thumbnail_id ) {
 			$image        = wp_get_attachment_image_src( $thumbnail_id, $small_thumbnail_size );
-			$image        = $image[0];
+			
+            if ( is_array( $image ) && isset( $image[0] ) ) {
+            	$image = $image[0];
+            } else {
+            	$image = false;
```

Comment:
Should we use the placeholder image instead of `false`?

```suggestion
            	$image = wc_placeholder_img_src();
```

This would guarantee the placeholder image is rendered instead of nothing.

Reply:
Anytime and thank you @Aljullu! Awesome, I'm glad you were able to reproduce the issue as well. Thanks for pointing out the linting requirement. Not sure how I missed that placeholder func, so thank you for pointing that out, good eye! I have made these updates :)